index.html

```diff
@@ -3168,8 +3167,11 @@ <h3>Semantic Annotations</h3>
 
     <p>
       As an example, the TD snippet given below extends the version information container
-      by addind version numbers for the hardware and firmware of the Thing,
-      semantically annotates a Property, and uses a well-defined value for the data schema unit.
+      by adding version numbers for the hardware and firmware of the <a>Thing</a>, and uses
+      values from external <a>Vocabularies</a> for the <a>Thing</a> and for the
+      data schema unit. SAREF [[smartM2M]] and OM, the Ontology of Units of Measure
```

Comment:
Suggest to make the links to <a>SAREF</a> and <a>OM</a> and change the other occurrences in the text.
Are SAREF and OM the *only* ontologies to be used for the home domain, or just examples?

Reply:
There is indeed an earlier occurrence of SAREF/OM at the end of Section 6.3.10. My suggestion is to remove these occurrences and simply point to Section 7.1 (as is already the case).

These vocabularies are examples (by design, everyone is free to expose their own vocabulary, including in the home automation domain). The words "among others" can be added at the end of the sentence.

Reply:
See a7ebc649e1430822a4d501ec1cde1874afa232a9 and cf53ebf4600df3119cdabc5acf1df2b5219199ed.